src/Airport.php

```diff
@@ -31,12 +32,21 @@ class Airport
      */
     private ?DirectDestinations $directDestinations;
 
+    /**
+     * <p>
+     *   A namespaced client for the
+     *   <code>/v1/airport/predictions</code> endpoints.
+     * </p>
+     */
+    private ?Predictions $predictions;
```

Comment:
No need to assign null  ```?``` on must existing endpoint ```$prediction```, see [PR ](https://github.com/amadeus4dev/amadeus-php/pull/73)